kubectl-rook-ceph.sh

```diff
@@ -263,6 +266,82 @@ function run_purge_osd() {
     rook ceph osd remove --osd-ids=$1 --force-osd-removal=$force_removal"
 }
 
+####################################################################################################
+# 'kubectl rook-ceph debug commands
+####################################################################################################
+
+function update_deployment_spec_command(){
+  deployment_spec=$1
+  main_command='["sleep","infinity"]'
+  deployment_spec=$(jq .template.spec.containers[0].command="$main_command" <<< $deployment_spec)
+  deployment_spec=$(jq .template.spec.containers[0].args=[] <<< $deployment_spec)
+}  
+
+function update_deployment_spec_image(){
+  deployment_spec=$1
+  alternalte_image=$2
+  deployment_spec=$(jq .template.spec.containers[0].image="$alternalte_image" <<< $deployment_spec)
```

Comment:
@BlaineEXE can you please comment the example about you were saying to return the value by `echo`

Reply:
```
function fun1(){
  echo 34
}

function fun2(){
  local res=$(fun1)
  echo $res
}
```

This way it is working,
Thanks

Reply:
https://github.com/rook/kubectl-rook-ceph/pull/36#discussion_r932425506